editor/plugins/polygon_2d_editor_plugin.cpp

```diff
@@ -275,7 +275,11 @@ void Polygon2DEditor::_uv_edit_mode_select(int p_mode) {
 		uv_button[UV_MODE_REMOVE_POLYGON]->hide();
 		uv_button[UV_MODE_PAINT_WEIGHT]->hide();
 		uv_button[UV_MODE_CLEAR_WEIGHT]->hide();
-		_uv_mode(UV_MODE_EDIT_POINT);
+		if (node->get_polygon().size() == 0) {
```

Comment:
```suggestion
		if (node->get_polygon().is_empty()) {
```

Reply:
Surrounding code uses `size() == 0`, but I agree `is_empty()` is better. The change has to start somewhere 😅